provisioning/transport/http/src/Generated/RuntimeRegistration.cs

```diff
@@ -170,7 +170,11 @@ public RuntimeRegistration(DeviceProvisioningServiceRuntimeClient client)
                 var ex = new HttpOperationException(string.Format(CultureInfo.InvariantCulture, "Operation returned an invalid status code '{0}'", _statusCode));
                 if (_httpResponse.Content != null)
                 {
+#if NET5_0
```

Comment:
Is this file really generated? We may not have any other choice than to hand edit it, but I worry what it'll mean when/if we regenerate.

Reply:
Yeah, so about this, it was auto-generated, but I couldn't locate the swagger for it in our repo. There also hasn't been any update on these files for over 2years.

Reply:
We might have generated it but might not have expected it to change (bad idea). For now, we've only giving it minimum possible updates.